kubernetes/structure_service_spec.go

```diff
@@ -136,6 +136,9 @@ func expandServiceSpec(l []interface{}) v1.ServiceSpec {
 	if v, ok := in["external_name"].(string); ok {
 		obj.ExternalName = v
 	}
+	if v, ok := in["external_traffic_policy"].(string); ok {
+		obj.ExternalName = v
```

Comment:
Is this corrrect - `obj.ExternalName` ?
Shouldn't that be something like `obj.ExternalTrafficPolicy` ?

Reply:
We'll also need to modify `flattenServiceSpec` similarly.